project-base/storefront/pages/index.tsx

```diff
@@ -33,11 +32,11 @@ const HomePage: FC<ServerSidePropsType> = () => {
                     <Banners />
                 </Webline>
                 <Webline className="mb-6">
-                    <Heading type="h2">{t('Promoted categories')}</Heading>
+                    <div className="h2 mb-3">{t('Promoted categories')}</div>
                     <PromotedCategories />
                 </Webline>
                 <Webline className="mb-6">
-                    <Heading type="h2">{t('Promoted products')}</Heading>
+                    <div className="h2 mb-3">{t('Promoted products')}</div>
```

Comment:
This change is also weird to me. To me those are important blocks which should have a heading, no?

Reply:
This is questionable. Since it is homepage, we can understand this as content of this page which can be relevant but information relevancy itself is zero. I am not considering it as relevant but now I when I think about different projects, they could be included with additional modules with headers which can be better relevant. That's why I've included headers here.